constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/utils/ViewSpline.java

```diff
@@ -1,81 +1,29 @@
-/*
- * Copyright (C) 2018 The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package androidx.constraintlayout.motion.widget;
+package androidx.constraintlayout.motion.utils;
 
 import android.os.Build;
-import androidx.constraintlayout.widget.ConstraintAttribute;
-import androidx.constraintlayout.motion.utils.CurveFit;
 import android.util.Log;
 import android.util.SparseArray;
 import android.view.View;
 
+import androidx.constraintlayout.core.motion.utils.CurveFit;
+import androidx.constraintlayout.motion.widget.Key;
+import androidx.constraintlayout.motion.widget.MotionLayout;
+import androidx.constraintlayout.core.motion.utils.SplineSet;
+import androidx.constraintlayout.widget.ConstraintAttribute;
+
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
-import java.text.DecimalFormat;
-import java.util.Arrays;
-
-/**
- * This engine allows manipulation of attributes by Curves
- *
- * @hide
- */
-
-public abstract class SplineSet {
-    private static final String TAG = "SplineSet";
-    protected CurveFit mCurveFit;
-    protected int[] mTimePoints = new int[10];
-    protected float[] mValues = new float[10];
-    private int count;
-    private String mType;
-
-    @Override
-    public String toString() {
-        String str = mType;
-        DecimalFormat df = new DecimalFormat("##.##");
-        for (int i = 0; i < count; i++) {
-            str += "[" + mTimePoints[i] + " , " + df.format(mValues[i]) + "] ";
 
-        }
-        return str;
-    }
-
-    public void setType(String type) {
-        mType = type;
-    }
+public abstract class ViewSpline extends SplineSet {
+    private static final String TAG = "ViewSpline";
```

Comment:
It doesn't look like `TAG` is used
```suggestion
```

Reply:
Log.e(TAG, "unable to setProgress", e);
 
                        Log.e(TAG, "unable to setProgress", e);